README.md

```diff
@@ -12,42 +12,77 @@ In your `flake.nix`:
 ```nix
 {
   # Use this repo as the `nixpkgs` URL
-  inputs.nixpkgs = "github:nix-ocaml/nix-overlays";
-
-
-  outputs = { self, nixpkgs }:
-    let
-      pkgs = nixpkg.packages.${"YOUR_SYSTEM_STRING"};
-    in
-    {
-      ...
-    };
+  inputs.nixpkgs.url = "github:nix-ocaml/nix-overlays";
+
+  outputs = {
+    self,
+    nixpkgs,
+  }: let
+    # Include supported systems e.g ["x86_64-linux" "aarch64-darwin"]
+    supportedSystems = [];
+
+    forAllSystems = f:
+      nixpkgs.lib.genAttrs supportedSystems
+      (system: f system (import nixpkgs {inherit system;}));
+
+    # Change 0_00 to a supported ocamlPackages version e.g 5_00 to get OCaml 5
+    ocamlPackagesFor = forAllSystems (system: pkgs: pkgs.ocaml-ng.ocamlPackages_0_00);
```

Comment:
let's just pick a version rather than having an invalid version. People can change it later if they want to, but at least they'll have a working flake.nix